### DISCUSSION TITLE
Dudas sobre el funcionamiento de DOUBLE-SORT-BY

### DISCUSSION THREAD

**Discussion by aangel-sama** [IIC2133-PUC/2024-2#107] [Tarea 1] [answered]
Tengo una duda de como funciona esto, porque por ejemplo, tengo los siguientes Pinguinos:
ID AGE M
8  90    1
5  88    0
6  59    1
0  31    0

En este caso si aplicamos el DOUBLE age desc membership asc, de es forma, asumo que se deben ordenar de la edad mas alta a la mas chica, pero siendo primeros los que no tienen membresía y luego los que si, por lo que el ordenamiento quedaría así:
5, 0, 8, 6

Sin embargo el ordenamiento correcto seria así:
8, 5, 6, 0

Y solo esta siendo ordenado por la edad desc, la membresía no tiene nada que ver ahí ya que, 8 tiene membresía y luego 5 no la tiene, 6 si, y 0 no, es como si fuera intercambiando. Por lo que, realmente no entiendo como funciona la función.

**Comment by Segismundios** [comment, marked as answer]
Hola @aangel-sama. El segundo orden se aplica sobre los pingüinos que coincidan en el primer atributo. En el ejemplo que das, el primer atributo es edad y no existen coincidencias entre pingüinos, por lo que el segundo criterio de orden no tiene efecto. 
Si existiera algún empate, por ejemplo, dos pingüinos tuvieran la misma edad, esos pingüinos los ordenaríamos según membresía.
Espero que haya quedado claro, saludos!

**Reply by aangel-sama** [reply]
Holaaa, sorry pero no entendí bien, podrías darme como un ejemplo?
